### ISSUE TITLE
Wordpress Binding

### ISSUE THREAD

**Issue by ghost** [arnecls/raidplaner#106]
tl:dr: Does Raidplaner need a plugin in the wordpress installation? I tryed "User Role Editor" - but it seems so that that doesn't work.

Raidplaner 1.1.3
Wordpress 4.0 (fresh installed)

While installing Raidplaner I have a problem with the Wordpress Binding. The Prefix in my Database for Wordpress is NOT "wp_" but f.e. "wpslakdjf_".

At the page Wordpress Binding on the right side I get the following error message:
"Warning: Invalid argument Supplied for foreach () in /<Path to the Raidplaner>/raidplaner/lib/private/bindings/wp.php on line 111". I added a echo for the sql-statement to show that the standard "wp_"-prefix is used:
![before1](https://cloud.githubusercontent.com/assets/9318897/4701472/81b9f8ca-5859-11e4-9bdb-e1416ba5ee19.png)

Then:
- I select Wordpress 3.x and check the [x] right beside it
- I clicked on the Button "Retrieve settings automatically" (that works)
  It seems that the page does not get reloaded after changing informations on the left side.
  ![after1](https://cloud.githubusercontent.com/assets/9318897/4701477/84e78fee-5859-11e4-8098-2bc5b77b8703.png)

I decided to press the button contine. That works. config.wp.php now has my wordpress binding safed (but with no role for raidmember/raidlead).

Again I switched to the bindings setup and now the correct prefix is shown.... But the same error message.
![after2](https://cloud.githubusercontent.com/assets/9318897/4701493/be776fe0-5859-11e4-9bde-c63b071ca368.png)

I took the statement to look in my wordpress database... but no records where found.

tl:dr: Does Raidplaner need a plugin in the wordpress installation? I tryed "User Role Editor" - but it seems so that that doesn't work.
